### ISSUE TITLE
Improvement for Parameters/Definitions

### ISSUE THREAD

**Issue by xiaoshir** [LCA-ActivityBrowser/activity-browser#455]
I think it would be nice to add two columns in parameter definitions: comment, unit. Whether the latter is considered in calculation is another question (i.e. users do not have to do unit conversion themselves).

**Comment by dgdekoning** [comment]
How do you present a parameter 'unit' to users?

Currently any implementation of these fields would be cosmetic only (with the comment having obvious use in remembering what the parameter is used for)

**Comment by xiaoshir** [comment]
Agree, if the unit is not used in the calculation,  then indeed it can be specified in just the comment field, for user‘s information
